core/webui/src/main/java/org/exoplatform/ecm/webui/utils/Utils.java

```diff
@@ -1067,7 +1067,7 @@ public static String getTitleWithSymlink(Node node) throws Exception {
       }
       if (nProcessNode.hasNode("jcr:content")) {

```

Comment:
Should we move your modification in the upper if ?
So that, if there is a exo:title, we do not have to read jcr:content subnode.
It requires less compute.

Reply:
Ok